pycbc/events/events.py

```diff
@@ -193,6 +193,20 @@ def newsnr(snr, reduced_x2, q=6., n=2.):
         return newsnr
     else:
         return newsnr[0]
+       
+def bluesnr(snr, bchisq, lchisq):
+    """ Combined SNR derived from NewSNR and Lat Chisq"""
```

Comment:
I think at least initially it should be. It's a lot safer to do so in the sense that we can more easily reconstruct what happened, i.e. we still have triggers to look at and follow up. It also allows us to more easily test different thresholds post-facto until we are happy to move to production (as fast as possible hopefully). When we are *really* comfortable with it and don't feel the need to every check it then depending on how we find to best incorporated it into a single detector statistic it could be done as a veto. It can't be done at the end of the code thought if that's what you mean. Not without regenerating a lot of intermediate products anyway.